packages/core-typings/src/IRoom.ts

```diff
@@ -192,6 +192,9 @@ export interface IOmnichannelGenericRoom extends Omit<IRoom, 'default' | 'featur
 
 export interface IOmnichannelRoom extends IOmnichannelGenericRoom {
 	t: 'l';
+	omnichannel?: {
+		predictedVisitorAbandonmentAt: Date;
```

Comment:
Shouldn't this be optional too? 👀 not sure if we add other props to "omnichannel" key.

Reply:
I checked this and didn't find any other prop being set inside "omnichannel", so I think we should be safe here :smiley: